src/events/acceleration.js

```diff
@@ -203,6 +244,18 @@ p5.prototype._handleMotion = function() {
     }
     old_max_axis = new_max_axis;
   }
+  var deviceShaken = this.deviceShaken || window.deviceShaken;
+  if (typeof deviceShaken === 'function') {
+    var accelerationChange = {};
```

Comment:
Looks like you're creating an object here but it's only being used internally to hold onto a few numbers? I think it'd be a little better to just store these as individual vars rather than wrapping up into one object as this would speed up performance (it's slower to create a new object vs creating a number). Alternatively, the accelerationChange var could be declared higher up outside this function, and reused inside the function.